apps/ios/LiveVideo/LiveVideo/Views/Participants/ParticipantsViewModel.swift

```diff
@@ -149,3 +153,9 @@ class ParticipantsViewModel: ObservableObject {
         viewerCount = viewersWithoutRaisedHand.count + viewersWithRaisedHand.count
     }
 }
+
+extension SyncUsersStore.User {
+    var displayName: String {
```

Comment:
Can this be `let` instead of `var` ?

Reply:
It cannot because it is a computed property. Only stored properties can be let. However because it is a stored property that is only read only it has the same exact behavior as a let in this case.